timeline.gd

```diff
@@ -68,6 +68,16 @@ func get_command(position:int) -> Resource:
 	push_error("get_command: Tried to get an command on a non-existing position: ", position)
 	return null
 
+## Get the command [code]position[/code] from its [code]bookmark[/code]
+func get_command_by_bookmark(bookmark:String) -> Resource:
```

Comment:
maybe we should add a `_bookmarks` dictionary that is built on load and updated on request to avoid iterating over all commands each time we want to get a certain bookmark?

Reply:
Good idea